lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -35,7 +39,7 @@ def get_all_dicom_treatment_info(dicomFile):
     except (TypeError, ValueError, AttributeError):
         prescriptionDescription = ""
 
-    for fraction in dicom.FractionGroupSequence:
+    for fraction in dicom[0x300A, 0x0070]:
```

Comment:
Is there a reason here to change from the Dicom Tag name attribute call? I find calling dicom files by their byte tags not as easy to know what is going on.

Reply:
It's because a while back I noticed that while the DICOM format has all the same tags, they aren't always nested the same way (BeamSequence might be within FractionGroupSequence or it might be outside). I was trying something here to see if it would work and just forgot to switch back for the time being. Essentially this only works with our Pinnacle exports as far as I'm aware and will crash with another RP file because of the nesting issues so I was trying to brainstorm a way to make it more universal!

Reply:
> BeamSequence might be within FractionGroupSequence or it might be outside

That's weird, I've never seen BeamSequence (https://dicom.innolitics.com/ciods/rt-plan/rt-beams/300a00b0) nested within FractionGroupSequence (https://dicom.innolitics.com/ciods/rt-plan/rt-fraction-scheme/300a0070) :/

Reply:
I may misremembering exactly what the issue was... I’ll look into it again on Monday! I just remember thinking that maybe the problem could be solved by calling tags instead is all :)

Reply:
Referenced Beam Sequence.
Which has some elements in common with Beam Sequence.
Delayed binding / loosely typed language s makes it easy to accidentally treat them almost the same.

Reply:
> Delayed binding / loosely typed language s makes it easy to accidentally treat them almost the same.

We certainly do have the option to create a typed DICOM wrapper around pydicom if we believe this is a sufficient issue...

Reply:
I think there are benefits to having duck typing for this, and attempting to access something that isn't there should show up quickly in an automated test, unless the automated test inappropriately mocks/populates elements that are not part of the sequence item.  So... no I think we can leave it as is.

Reply:
It actually seems the issue was much simpler than I remembered... There just seems to be a Tolerance sequence missing from our institution's Raystation RP files that wasn't in the Pinnacle RP files.